WTWSMS/events/WtWSMS_holy_orders_events.txt

```diff
@@ -86,18 +86,19 @@ narrative_event = {
 			create_character = {
 				random_traits = yes
 				dynasty = random
-				religion = semitic_pagan_reformed
+				religion = semitic_pagan
```

Comment:
@loup99 

Why in the HECK should he follow a reformed religion that doesn't exist?!